pyproject.toml

```diff
@@ -3,6 +3,7 @@ name = "navigator_backend"
 version = "1.8.2"
 description = ""
 authors = ["CPR-dev-team <tech@climatepolicyradar.org>"]
+packages = [{ include = "app" }, { include = "tests" }]
```

Comment:
I don't this is technically necessary as both packages exist at the project root.

Reply:
It unfortunately is very necessary for local development, without this, you will get import errors using Trunk with a virtual environment (whether that's a venv or a pyenv)